packages/metadata-service/src/readers/abebooks.service/index.ts

```diff
@@ -67,6 +81,10 @@ export class AbeBooksService {
 
   async getData(isbn: string): Promise<Bundle> {
     const result = await this.readData(isbn);
-    return this.parseData(result);
+    const img = await this.digitalSpaceService.createFile({
+      fileExtension: result.url.imageUrl.split('.').pop(),
+      buffer: result.img,
+    });
+    return this.parseData(result, img);
```

Comment:
It's make more sense not to override `parseData`, but instead assign image here in this function.